packages/tools/devtools/devtools-example/webpack.config.cjs

```diff
@@ -20,6 +20,11 @@ module.exports = (env) => {
 				main: path.join(sourcePath, "index.tsx"),
 			},
 			resolve: {
+				extensionAlias: {
+					".cjs": [".cts", ".cjs"],
+					".js": [".ts", ".tsx", ".js"],
+					".mjs": [".mts", ".mjs"],
+				},
```

Comment:
Can you clarify what this is doing / why was it necessary? I'm trying to understand it but not finding webpack's docs or quick searches very fruitful.